lib/kennel/syncer.rb

```diff
@@ -79,7 +79,7 @@ def calculate_diff
               fill_details(a, cache) if e.class::API_LIST_INCOMPLETE
               diff = e.diff(a)
               @update << [id, e, a, diff] if diff.any?
-            elsif tracking_id(a) # was previously managed
+            elsif tracking_id(a) && !@lookup_map[tracking_id(a)] # was previously managed (need to check lookup map in case of dashboards)
```

Comment:
why can't we put the tracking-id into the description ?

Reply:
this check is to make sure that not only is the tracking id present, but that it's not present in the lookup map for a different type. this is because "dash" will be present for dashboards, but we don't want to delete that because it will get used/matched later on